paasta_tools/paasta_maintenance.py

```diff
@@ -567,33 +576,31 @@ def paasta_maintenance():
     duration = args.duration
 
     if action == 'drain':
-        return drain(hostnames, start, duration)
+        ret = drain(hostnames, start, duration)
     elif action == 'undrain':
-        return undrain(hostnames)
+        ret = undrain(hostnames)
     elif action == 'down':
-        return down(hostnames)
+        ret = down(hostnames)
     elif action == 'up':
-        return up(hostnames)
+        ret = up(hostnames)
     elif action == 'status':
-        return status()
+        ret = "%s" % status()
     elif action == 'schedule':
-        ret = schedule()
-        print "%s" % ret
-        return ret
+        ret = "%s" % schedule()
     elif action == 'is_safe_to_kill':
-        return is_safe_to_kill(hostnames[0])
+        ret = is_safe_to_kill(hostnames[0])
     elif action == 'is_host_drained':
         ret = is_host_drained(hostnames[0])
-        print ret
-        return ret
     elif action == 'is_host_down':
-        return is_host_down(hostnames[0])
+        ret = is_host_down(hostnames[0])
     elif action == 'is_host_draining':
-        return is_host_draining(hostnames[0])
+        ret = is_host_draining(hostnames[0])
     elif action == 'is_host_past_maintenance_start':
-        return is_host_past_maintenance_start(hostnames[0])
+        ret = is_host_past_maintenance_start(hostnames[0])
     elif action == 'is_host_past_maintenance_end':
-        return is_host_past_maintenance_end(hostnames[0])
+        ret = is_host_past_maintenance_end(hostnames[0])
+    print ret
```

Comment:
fair point. I'll remove the printing for anything that's just an exit code before I ship